From 23c9c896d7d03c6d3f6cebb44d671be9219883a6 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Mon, 16 Jul 2018 17:09:57 +0800
Subject: [PATCH 4163/5242] MLK-18922-1 ARM: arm64: fsl-imx8mq-evk: add
 disable OTG capabilites

commit  c18f3affe7c534dfc775ca6ff5656a535c8e9c4f from
https://source.codeaurora.org/external/imx/linux-imx.git

imx850d evk has no OTG capabilities, so we disable all OTG capabilities
at dts.

Acked-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index e96c480..73ce4db 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -715,6 +715,9 @@
 	status = "okay";
 	extcon = <&typec_ptn5100>;
 	dr_mode = "otg";
+	hnp-disable;
+	srp-disable;
+	adp-disable;
 };
 
 &usb3_phy1 {
-- 
1.7.9.5

